components/Backgrounds/Background.styled.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export const TrackerFormBackground = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: start;
+  gap: 1rem;
+  padding-top: 1rem;
+  height: 100vh;
+  width: 100vw;
```

Comment:
By default, block level elements will take up the full width of the page (depending on other styling). I "think" in this case we do not need to declare this.